dashboard/test/models/hint_view_request_test.rb

```diff
@@ -0,0 +1,58 @@
+require 'test_helper'
+
+class HintViewRequestTest < ActiveSupport::TestCase
+
+  setup do
+    @student = create :student
+    @script = create :script
+    @level_one = create :level
+    @level_two = create :level
+  end
+
+  test "can retrieve feedback for milestone request" do
+    HintViewRequest::ENABLED = true
+
+    HintViewRequest.create(
+      script: @script,
+      level: @level_one,
+      user: @student,
+      feedback_type: 1,
+      feedback_xml: '<block type="first"></block>'
+    )
+    HintViewRequest.create(
+      script: @script,
+      level: @level_one,
+      user: @student,
+      feedback_type: 1,
+      feedback_xml: '<block type="second"></block>'
+    )
+    HintViewRequest.create(
+      script: @script,
+      level: @level_two,
+      user: @student,
+      feedback_type: 1,
+      feedback_xml: '<block type="third"></block>'
+    )
+
+    first_level_response = HintViewRequest.milestone_response(@script, @level_one, @student);
+    second_level_response = HintViewRequest.milestone_response(@script, @level_two, @student);
+
+    assert_equal first_level_response, [{feedback_type: 1, feedback_xml: '<block type="first"></block>'}, {feedback_type: 1, feedback_xml: '<block type="second"></block>'}]
```

Comment:
Could potentially pull each of the feedback `{}`s out into variables for line length / scan-ability